style.css

```diff
@@ -13,10 +14,338 @@ h1,h2,h3,h4,h5,h6{
+}
+.index__logo_second{
+    padding-top: 83px;
+}
+.container__card{
+    padding-left: 180px;
+    padding-top: 123px;
+}
+.card__inner{
+    display: flex;
+    justify-content: space-around;
+    text-align: center;
+}
+
+.card_title{
+    font-size: 24px;
+    line-height: 36px;
+    text-align: center;
+    letter-spacing: -0.750316px;
+    color: #19191B;
+    text-align: center;
+    padding-top: 7px;
+    padding-bottom: 10px;
+}
+.card_about{
+    font-size: 16px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: -0.500211px;
+    color: #696871;
+    text-align: center;
+
+}
+.card__first{
+    padding-right: 55px;
+}
+.card__first_thirrd{
+    margin-right: 175px;
+}
+.footer__container{
+    margin-left: 165px;
+    padding-top: 130px;
+
+}
+.footer__title{
+    display: flex;
+    justify-content: space-between;
+}
+.footer_btn{
+    margin-right: 183px;
+    padding-top: 50px;
+}
+.footer_name{
+    width: 523px;
+    font-size: 60px;
+    line-height: 70px;
+    letter-spacing: -1.87579px;
+    color: #19191B;
+}
+.first_foot_bnt{
+    color: white;
+    background-color:#5454D4 ;
+    padding: 18px 7px 16px 8px;
+    border: none;
+    border-radius: 10px 0px 0px 10px;
+    width: 160px;
+    font-size: 16px;
+    line-height: 21px;
+    text-align: center;
+    letter-spacing: -0.500211px;
+    font-family: 'DM Sans', sans-serif;
+}
+.second_foot_bnt{
+    font-size: 16px;
+    line-height: 21px;
+    text-align: center;
+    letter-spacing: -0.500211px;
+    color: #19191B;
+    font-family: 'DM Sans', sans-serif;
+    background-color: #F7F7FB; 
+    border: none;
+    border-radius: 0px 10px 10px 0;
+    width: 160px;
+    padding: 18px 7px 16px 8px;
+}
+.footer__card_container{
+    padding-top: 98px;
+    padding-left: 185px;
+}
+.footer__card{
+    display: flex;
+    justify-content: baseline;
+}
+.first__footer_card{
+    width: 330px;
+    height: 441px;
+    border: 1px solid black;
+    text-align: center;
+    background-color: #F8F8F8;
+    border-radius: 5px;
+    margin-right: 18px;
+}
+.name__footer_card{
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    letter-spacing: -0.562737px;
+    color: #696871;
+    padding-top: 14px;
+    padding-bottom: 7px;
+}
+.title_footer{
+    font-size: 60px;
+    line-height: 56px;
+    text-align: center;
+    letter-spacing: -1.03448px;
+    color: #1D293F;
+}  
+
+.about_footer{
+    font-size: 17px;
+    line-height: 50px;
+    text-align: center;
+    letter-spacing: -0.531474px;
+    color: #696871;
+    font-family: 'Mulish', sans-serif;
+    padding: 5px 106px 36px 106px;
+    width: 136px;
+}
+.first__footer_card.second{
+    background-color: #FF7143;
+    width: 376px;
+    height: 473px;
+    position: relative;
+    bottom: 20px;
+    
+}
+.name__footer_card2{
+    font-size: 18px;
+    line-height: 28px;
+    text-align: center;
+    letter-spacing: -0.562737px;
+    color: rgba(255, 255, 255, 0.7);
```

Comment:
you should write color in one style
if write color as HEX, write it

Reply:
Not ready.
How to set transparency in HEX format?

Reply:
transparency  don't exist in HEX format